dev-proxy-plugins/Inspection/DevToolsPlugin.cs

```diff
@@ -83,6 +102,25 @@ private void InitInspector()
         webSocket.Start();
 
         var processName = GetChromiumProcessName(_configuration);
+
+        // check if the browser is installed
+        if (!File.Exists(processName))
+        {
+            Logger.LogError("Browser executable not found at {processName}", processName);
+            return;
+        }
+
+        // find if the process is already running
+        var processes = Process.GetProcessesByName(Path.GetFileNameWithoutExtension(processName));
+
+        if (processes.Length > 0)
+        {
+            Logger.LogError(@"Found existing process {processName} with Id {processId}. 
+            Could not start DevTools plugin. 
```

Comment:
Let's replace the `DevTools plugin` string with the value of the `Name` property for consistency.